.github/workflows/build-and-install.yml

```diff
@@ -59,7 +59,7 @@ jobs:
             rmjsonc: 'apk del json-c-dev'
 
           - distro: 'archlinux:latest'
-            pre: 'pacman --noconfirm -Sy grep libffi'
+            pre: 'pacman --noconfirm -Syu && pacman --noconfirm -Sy grep libffi'
```

Comment:
If you use `-Syyu` it will upgrade the package lists if necessary.

Reply:
Not quite. `-yy` forces a refresh of the lists, just `-y` will do one if needed, which will always happen here because Arch strips the package cache from their images before the publish them.